pkg/types/slice.go

```diff
@@ -17,8 +17,3 @@ func Remove(haystack []string, needle string) []string {
 	}
```

Comment:
`slices` has a `Contains` and `DeleteFunc` functions, so we can remove the `Contains` and `Remove` functions above.

Reply:
Thanks, I deleted these but they got re-added in a bad merge 🤦‍♂️